userguide/system.rst

```diff
@@ -1132,17 +1132,23 @@ These settings are described in
    |                     |           | data sent by the Graphite plugin.                   |
    |                     |           |                                                     |
    +---------------------+-----------+-----------------------------------------------------+
-   | Graph Age           | integer   | Maximum age a graph is stored in months.            |
+   | Graph Age           | integer   | Maximum age a graph is stored in months. Changing   |
+   |                     |           | the value requires that the reporting database be   |
```

Comment:
Not clear what "requires the reporting database be destroyed" means here.  From later reading, it enables the "Confirm RRD Destroy" checkbox, and changes only take effect after that. So maybe:
```
Changing this value causes the :guilabel:`Confirm RRD Destroy` checkbox to appear.
Changes do not take effect until the existing reporting database is destroyed.
```